src/render-html/messageAsHtml.js

```diff
@@ -72,12 +112,36 @@ const messageBody = ({
   reactions: ReactionType[],
   realmEmoji: ReactionType,
   timeEdited: Date,
-}) => `
+}) => {
+  let val = null; 
+  // Regex to scan out image URLs
+  const regex = /<img src="(.*)">/g; 
+  // A replacement string to convert relative to full URL manually and replace it within img tags
+  const substr = `<img src="https://hsinter0.zulipchat.com/static$1" >`; 
```

Comment:
Replace img urls to full domain urls

Reply:
Are you sure the `baseUrl` parameter added to the webview does not fix that automatically?
It will be good to confirm.

Reply:
In reality I am not able to confirm that because due to auth headers issue no images were loaded. As a result I can't verify if `baseUrl` works a we are expecting. Earlier I thought that `baseUrl` is not working so I decided to test embedding full Url into HTML but that also failed. I guess this question could be easily answered once we are able to authenticate with server.